File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Chrome()
wait = WebDriverWait(driver, 10)

driver.maximize_window()
driver.get("https://training-support.net/webelements/dynamic-content")
print("Title of page is :", driver.title)

click_me = driver.find_element(By.ID, "genButton")
text = driver.find_element(By.ID, "word")

click_me.click()

if wait.until(EC.text_to_be_present_in_element((By.ID, "word"), "release")):
    print("Text Release found")

driver.quit()

File: Selenium/Activities/Activity11.java
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;
import java.time.Duration;

public class Activity11 {

    public static void main(String[] args) {

        ChromeDriver driver = new ChromeDriver();
        WebDriverWait wait = new WebDriverWait(driver,Duration.ofSeconds(10));
        
        driver.manage().window().maximize();

        driver.get("https://training-support.net/webelements/dynamic-content");
        System.out.println("Title of page is : " + driver.getTitle());

        WebElement clickMe = driver.findElement(By.id("genButton"));
        WebElement text = driver.findElement(By.id("word"));
        clickMe.click();
       if(wait.until(ExpectedConditions.textToBePresentInElement(text, "release"))) {
    	   System.out.println("Text Release found");
       }
        
        
        driver.quit();
    }
}
